projects/ui/src/shim.cds-core.scss

```
@@ -18,6 +18,20 @@
 }
 
 $metropolis-font-family: Metropolis, 'Avenir Next', 'Helvetica Neue', Arial, sans-serif;
+/*
+ * Overrides cds-text font-weights due to disability to bump core peer dependency version for ng-clarity v16
+ * and current version being v6.7.0 while this was updated in v6.8.0 https://github.com/vmware-clarity/core/pull/276
+ */
+[cds-text],
+[cds-theme] {
+  --cds-global-typography-body-font-weight: 500;
+  --cds-global-typography-display-font-weight: 500;
+  --cds-global-typography-heading-font-weight: 500;
+  --cds-global-typography-headline-font-weight: 500;
+  --cds-global-typography-title-font-weight: 500;
+  --cds-global-typography-section-font-weight: 500;
+  --cds-global-typography-subsection-font-weight: 500;
```

The typography values are available as both globals and aliases. 
```suggestion
  --cds-global-typography-subsection-font-weight: 500;
  --cds-alias-typography-body-font-weight: 500;
  --cds-alias-typography-display-font-weight: 500;
  --cds-alias-typography-heading-font-weight: 500;
  --cds-alias-typography-headline-font-weight: 500;
  --cds-alias-typography-title-font-weight: 500;
  --cds-alias-typography-section-font-weight: 500;
  --cds-alias-typography-subsection-font-weight: 500;
```

---

Thanks!